eng/targets/Imports.targets

```diff
@@ -175,8 +175,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <!-- Include BannedSymbols covering all projects -->
-    <AdditionalFiles Include="$(MSBuildThisFileDirectory)..\config\BannedSymbols.txt" />
+    <!-- Include BannedSymbols covering all product projects -->
+    <AdditionalFiles Include="$(MSBuildThisFileDirectory)..\config\BannedSymbols.txt" Condition="'$(IsTestProject)' == 'false'" />
```

Comment:
I guess this is fine if it works, but the normal process for msbuild props would be `'$(IsTestProject)' != 'true'` to allow for empty strings

Reply:
You're right. Will fix.